lib/msal-common/src/authority/TrustedAuthority.ts

```diff
@@ -0,0 +1,93 @@
+import { Constants } from "../utils/Constants";
+import { INetworkModule } from "../network/INetworkModule";
+import { ICloudInstanceDiscoveryResponse } from "./ICloudInstanceDiscoveryResponse";
+import { CloudDiscoveryMetadataType } from "./CloudDiscoveryMetadataType";
+import { ICloudDiscoveryMetadata } from "./ICloudDiscoveryMetadata";
+import { ClientConfigurationError } from "../error/ClientConfigurationError";
+
+export class TrustedAuthority {
+    private static CloudDiscoveryMetadata: CloudDiscoveryMetadataType = {};
```

Comment:
do we need to make these static? can we make these all member functions of the prototype instead?

Reply:
Yes because we never instantiate `TrustedAuthority` this variable should only be set once